cmd/runner/runner.go

```diff
@@ -107,13 +107,13 @@ func main() {
 				if *buildEndpoints {
 					err := doBuildEndpoints(client, server, *verbose)
 					if err != nil {
-						fmt.Println(err)
+						log.Printf("ERROR: %s", err.Error())
```

Comment:
If I understand correctly, this is the standard logger and just writes to `stderr` correct? We could also just replace the `log.Printf() + os.Exit(1)` sequence with `log.Fatal{f}`.

Reply:
Yes, it is just the standard logger. I can change it to fatal on the appropriate places ;)